src/WemoSwitch.cpp

```diff
@@ -9,7 +9,7 @@ WemoSwitch::WemoSwitch(){
 }
 //WemoSwitch::WemoSwitch(String alexaInvokeName,unsigned int port){
 WemoSwitch::WemoSwitch(String alexaInvokeName, unsigned int port, CallbackFunction oncb, CallbackFunction offcb){
-    uint32_t chipId = ESP.getChipId();
+    uint32_t chipId = ESP.getChipId() + port;
```

Comment:
small comment but I would change the variable name to reflect its the chipIdWithPort